### DISCUSSION TITLE
Is is possible to configure the SEB Browser to use a default start url instead of connecting with an LMS?

### DISCUSSION THREAD

**Discussion by jorritpr** [SafeExamBrowser/seb-win-refactoring#501] [Q&A]
Hi,

Is it possible to configure the SEB browser to start with a pre-defined url and circumvent the part where you have to connect to an LMS? Preferably in a way that the browser fetches it's configuration from the seb-server each time it starts, so we can better manage and configure it. 

All attempts we have made so far seem to require the LMS connection which we cant use, our LMS (Brightspace) isn't supported unfortunately. Next to that, we already have a "exam entrance" webpage where they can log in and take their exam, where we can disable access to the other courses, so it isn't really necessary for us to use an LMS connection.

Hope someone knows whether this is possible! :)

kind regards,

Jorrit

**Comment by heinzkrebs** [comment]
servas,

i've configured my seb-configuration file with a start url that points to my examination server. so if the candidate opens that seb-file he/she gets directly connected to the login page of the server.

this seb-file is distributed by email to the candidates, but also put on the webserver, so they can just visit https://examserver/exam.seb, the seb-file will be downloaded and the seb browser will start using this configuration.

i don't use any LMS ....

does this answer your question?

ciao
  H.

**Comment by dbuechel** [comment]
You might also find some further useful information under https://safeexambrowser.org/developer/seb-integration.html.
